src/mono/mono.proj

```diff
@@ -856,7 +856,7 @@
 
     <PropertyGroup>
       <_MonoSkipInitCompiler Condition="'$(RealTargetArchitecture)' != '' and '$(RealTargetArchitecture)' != '$(BuildArchitecture)'">false</_MonoSkipInitCompiler>
-      <_MonoAotCrossOffsetsCommand Condition="'$(MonoUseCrossTool)' == 'true'">$(PythonCmd) $(MonoProjectRoot)mono/tools/offsets-tool/offsets-tool.py @(MonoAotCrossOffsetsToolParams, ' ')</_MonoAotCrossOffsetsCommand>
+      <_MonoAotCrossOffsetsCommand Condition="'$(MonoUseCrossTool)' == 'true'">python $(MonoProjectRoot)mono/tools/offsets-tool/offsets-tool.py @(MonoAotCrossOffsetsToolParams, ' ')</_MonoAotCrossOffsetsCommand>
```

Comment:
This should still use `$(PythonCmd)`, so that it works on other platforms.

You can change it conditionally to `python` on win/arm64.

Reply:
changed and tested with WASM CI for Arm64
https://github.com/Windows-on-ARM-Experiments/runtime/actions/runs/4798720517